src/test/java/com/epam/tc/hw5/ex1/steps/AssertionStep.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.tc.hw5.ex1.steps;
+
+import io.cucumber.java.en.Then;
+
+public class AssertionStep extends AbstractStep {
```

Comment:
you don't have to create separated AssertionStepS classes for each exercise. fix it.